x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/field_parameters/select_inference_id.tsx

```diff
@@ -135,14 +144,29 @@ export const SelectInferenceId = ({
         setIsInferenceFlyoutVisible(!isInferenceFlyoutVisible);
         setIsCreateInferenceApiLoading(false);
         setInferenceAddError(undefined);
+        const getTrainedModelId = () => {
+          if (modelConfig.service === Service.elser) return ElasticsearchModelDefaultOptions.elser;
```

Comment:
We can use switch-case here to make the code more readable:

```
const getTrainedModelId = () => {
  switch (modelConfig.service) {
    case Service.elser:
      return ElasticsearchModelDefaultOptions.elser;
    case Service.elasticsearch:
      return ElasticsearchModelDefaultOptions.e5;
    default:
      return '';
  }
};
```

OR, we can create a map to make it concise:

```
const getTrainedModelId = () => {
  const serviceMap: Record<string, string> = {
    [Service.elser]: ElasticsearchModelDefaultOptions.elser,
    [Service.elasticsearch]: ElasticsearchModelDefaultOptions.e5,
  };

  return serviceMap[modelConfig.service] || '';
};
```

Reply:
updated in https://github.com/elastic/kibana/pull/183869/commits/838e297be43e0dbbfb242161eea1a4a4fe346524